main/test/flix/CompilerSuite.scala

```diff
@@ -142,7 +142,8 @@ class CompilerSuite extends FlixSuite(incremental = true) {
   //
   // Effects
   //
-  mkTest("main/test/flix/Test.Exp.Effect.flix")
+  // TODO temporarily allowing redundancies during development
+  mkTest("main/test/flix/Test.Exp.Effect.flix")(options.copy(xallowredundancies = true))
```

Comment:
Nina will soon come after you :)

Reply:
Guess I better hurry with the development!